actionpack/lib/action_dispatch/middleware/templates/rescues/actionable_error.html.erb

```diff
@@ -0,0 +1,65 @@
+<header>
+  <h1>
+    <%= @exception.class.to_s %>
+    <% if @request.parameters['controller'] %>
+      in <%= @request.parameters['controller'].camelize %>Controller<% if @request.parameters['action'] %>#<%= @request.parameters['action'] %><% end %>
+    <% end %>
+  </h1>
+</header>
+
+<div id="container">
+  <h2>
+    <%= h @exception.message %>
```

Comment:
Why do we need `html_escape` here? I thought Rails did it automatically?

Reply:
Sorry for the late chime-in. I copied the template from the other and haven't given it a thought. 😅 Could be a "historical" (not to be confused with legacy) reason. I will check!